libretroshare/src/pqi/authssl.cc

```diff
@@ -47,13 +47,13 @@
 
 /* SSL connection diagnostic */
 
-const uint32_t RS_SSL_HANDSHAKE_DIAGNOSTIC_UNKNOWN                     = 0x00 ;
+//const uint32_t RS_SSL_HANDSHAKE_DIAGNOSTIC_UNKNOWN                     = 0x00 ;
```

Comment:
Please keep these 3. Maybe they are not used for now but they should in the future when we add more feedback on certificates. Also the first one is here to have a name to uninitialized/default values.

Reply:
They stay in comment so you can uncomment them when needed.
They are const not enum.